Description: Upstream changes introduced in version 0.0.1+svn20110409.0930-5yavdr1~oneiric
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 eventlircd (0.0.1+svn20110409.0930-5yavdr1~oneiric) oneiric; urgency=medium
 .
   * link explicit against librt
 .
 The person named in the Author field signed this changelog entry.
Author: Gerald Dachs <gda@dachsweb.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- eventlircd-0.0.1+svn20110409.0930.orig/src/Makefile.am
+++ eventlircd-0.0.1+svn20110409.0930/src/Makefile.am
@@ -2,7 +2,7 @@ sbin_PROGRAMS = eventlircd
 eventlircd_SOURCES = main.c monitor.c monitor.h input.c input.h lircd.c lircd.h
 eventlircd_CFLAGS = $(AM_CFLAGS) $(LIBUDEV_CFLAGS) -DLIRCD_SOCKET=\"$(LIRCD_SOCKET)\" -DEVMAP_DIR=\"$(EVMAP_DIR)\"
 eventlircd_LDFLAGS = $(AM_CFLAGS)
-eventlircd_LDADD = $(LIBUDEV_LIBS)
+eventlircd_LDADD = $(LIBUDEV_LIBS) -lrt
 
 nodist_eventlircd_SOURCES = event_name_to_code.h evkey_code_to_name.h evkey_type.h
 
